### ISSUE TITLE
Pop up an error after creating a new storage account with "Microsoft RRDC Lab 1" location

### ISSUE THREAD

**Issue by v-xinda** [microsoft/vscode-azurestorage#1013]
**OS:** All
**Build:** 20220119.2
**Regression:** Not a regression

**Repro Steps:**
1. F1 -> Azure Storage: Create Storage Account...(Advanced) -> Enter a name -> Create a new resource group -> Yes -> Enter default value for index document and 404 error document -> Select **Microsoft RRDC Lab 1** location.
2. Check whether doesn't pop up an error after creating a new storage account.

**Expect:**
Don't pop up an error after creating a new storage account.

**Actual:**
Pop up an error after creating a new storage account.
![image](https://user-images.githubusercontent.com/57662131/150263569-900f7b9c-74d1-4f8d-8562-880e2748e782.png)
Output Details:
```WARNING: Provider "Microsoft.Resources" does not support location "East US 2 EUAP". Using "East US 2 EUAP" instead.
Creating resource group "anicoleedge" in location "East US 2 EUAP"...
Successfully created resource group "anicoleedge".
Creating storage account "anicoleedge" in location "microsoftrrdclab1" with sku "Premium_LRS"...
Successfully created storage account "anicoleedge".
Error: XML specified is not syntactically valid.
RequestId:e16054fb-d01c-0004-3fa3-0d09ff000000
Time:2022-01-20T02:13:44.6286339Z
```
**More Info:**
1. This issue doesn't reproduce for other locations.
2. This issue doesn't reproduce after selecting  "No" when asking to enable static website hosting.

**Comment by alexweininger** [comment]
I'm in contact with the EdgeZone team and they've told me that BlockBlob storage accounts are not available in the "East US EUAP" environment. I'm awaiting response if this is specific to "East US EUAP" or not.

**Comment by alexweininger** [comment]
Fixed in #1017
